.github/workflows/publish-storybook.yml

```diff
@@ -20,10 +20,10 @@ jobs:
           cache: yarn
 
       - run: yarn install --frozen-lockfile
-      - run: yarn workspace @itwin/itwinui-react build-storybook
+      - run: yarn build
```

Comment:
sure we need to build everything now?

Reply:
Added `--filter=storybook`

Reply:
I remember we had some issues in CI where we missed the `build` command.
Dont remmeber details now, but if you can check that this is not influenced right now, would be good

Reply:
oh.. build included copying at first (readme), but then it failed, cause it did not include changelog or something :D so we changed to copy files.. 
I dont remember truly :D